<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HW_Train_Controller</class>
 <widget class="QMainWindow" name="HW_Train_Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="DoorButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Doors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TempButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Temperature</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AdsButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>117</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Turn on advertiements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnnounceButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Announce</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModeButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>290</y>
      <width>117</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BrakeButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>117</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Change Brake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FailureButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Failures</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SpeedButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>560</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PowerButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KpKiButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>560</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Kp and Ki</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PEBrakebutton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>560</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Passenger Emergency Brake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EBrakeButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>560</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency Brake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Logout">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="DoorLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdsLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TempLCD">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="SpeedLCD">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="KpLCD">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>452</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="KiLCD">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>512</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="AnnounceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="SignalFailure">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="EngineFailure">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="BrakeFailure">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="EBrakeLabel">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ModeLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="PEBrakeLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="PowerLCD">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="BrakeProgress">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../../../../../Desktop/Sys and Proj/Sys and proj.qrc"/>
 </resources>
 <connections/>
</ui>
